css/carousel.css

```diff
@@ -1,32 +1,70 @@
-.full-image {
-    display: none;
+section#carousel {
+  width: 90%;
+  height: 30vh;
+  border-radius: 15px;
+  box-shadow: 2px 2px 4px rgba(141, 140, 140, 0.795);
+  position: relative;
+  margin-bottom: var(--spacer-5);
+  margin-top: var(--spacer-5);
 }
 
-.displayed {
-    display: block;
-    object-fit: cover;
-    width: 70%;
-    margin-bottom: 30px;
-    box-shadow: 1px 2px 3px grey;
-    border-radius: 15px;
+.carousel {
+  object-fit: cover;
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  border-radius: 15px;
+  overflow: hidden;
+  background: black;
+  opacity: 0.5;
 }
 
-.thumbnails{
-    display: grid;
-    grid-template-columns: repeat(4, 1fr);
-    width: 95vw;
-    grid-gap: 10px;
-    margin-bottom: 20px;
-    object-fit: cover;
-    border-radius: 15px;
+#carousel > .carousel > .image-container > h2 {
```

Comment:
it could be hard to maintain the changes for these layers. Logic could be 
#carousel h2 (style applies for all h2 elements) or something similar